src/test/java/runners/FormsTest.java

```diff
@@ -0,0 +1,30 @@
+package runners;
+
+import io.cucumber.junit.Cucumber;
+import io.cucumber.junit.CucumberOptions;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.runner.RunWith;
+import utils.BrowserManagement;
+
+
+@RunWith(Cucumber.class)
+@CucumberOptions(
+        plugin = {"pretty", "html:target/cucumberHtmlReport"},
+        features = "src/test/resources/features/forms.feature",
+        glue = {"steps"}
+)
+class FormsTest {
+
+    @BeforeClass
+    public static void init() {
+        BrowserManagement.startServer();
+    }
+
+    @AfterClass
+    public static void finish() {
+        BrowserManagement.getDriver().closeApp();
+        BrowserManagement.getDriver().quit();
+        BrowserManagement.stopServer();
+    }
+}
```

Comment:
These methods will be repeated in different tests. Maybe you can make this more abstract so you don't repeat code

Reply:
An abstract class is created for runners